src/graphql/queries/__tests__/__snapshots__/ListCooccurrences.js.snap

```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ListArticles lists all cooccurrences 1`] = `
+Object {
+  "data": Object {
+    "ListCooccurrences": Object {
+      "edges": Array [
+        Object {
+          "cursor": "WyJsaXN0Q29vY2N1cnJlbmNlVGVzdDIiXQ==",
+          "node": Object {
+            "id": "listCooccurrenceTest2",
+          },
+        },
+        Object {
+          "cursor": "WyJsaXN0Q29vY2N1cnJlbmNlVGVzdDEiXQ==",
+          "node": Object {
+            "id": "listCooccurrenceTest1",
+          },
+        },
+      ],
+      "pageInfo": Object {
+        "firstCursor": "WyJsaXN0Q29vY2N1cnJlbmNlVGVzdDIiXQ==",
+        "lastCursor": "WyJsaXN0Q29vY2N1cnJlbmNlVGVzdDEiXQ==",
+      },
+      "totalCount": 2,
+    },
+  },
+}
+`;
+
+exports[`ListArticles sorts: by updatedAt DESC 1`] = `
+Object {
+  "data": Object {
+    "ListCooccurrences": Object {
+      "edges": Array [
+        Object {
+          "node": Object {
+            "id": "listCooccurrenceTest2",
+            "updatedAt": "1",
+          },
+        },
+        Object {
+          "node": Object {
+            "id": "listCooccurrenceTest1",
+            "updatedAt": "1",
```

Comment:
nit: the two `updatedAt` are the same, cannot see if sorting is really applied lol

Reply:
updated